Doc/using/cmdline.rst

```diff
@@ -861,16 +863,17 @@ conflict.
    Python memory allocations using the :mod:`tracemalloc` module. The value of
    the variable is the maximum number of frames stored in a traceback of a
    trace. For example, ``PYTHONTRACEMALLOC=1`` stores only the most recent
-   frame. See the :func:`tracemalloc.start` for more information.
+   frame. See the :func:`tracemalloc.start` for more information. This is
+   equivalent to setting the :option:`-X` ``tracemalloc`` option.
```

Comment:
```suggestion
   frame. See the :func:`tracemalloc.start` for more information.
   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```

Reply:
Fix grammar issue (from the original, but on a modified line): either

```suggestion
   frame. See :func:`tracemalloc.start` for more information.
   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```

or

```suggestion
   frame. See the :func:`tracemalloc.start` function for more information.
   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```